backend/src/main/scala/cromwell/backend/BackendActor.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.backend
+
+import akka.actor.{ActorRef, Actor, ActorLogging}
+import akka.event.LoggingReceive
+import cromwell.backend.BackendActor.{Validate, AfterAll, BeforeAll, CreateJobExecutor}
+import cromwell.backend.model.JobDescriptor
+
+object BackendActor {
```

Comment:
Hmm, I agree with your comments but I think there's a middle ground here. For example, to me the name `BackendActor` implies that it's shared between Workflows. But if I understand correctly you get a new BackendActor per backend per workflow?